packages/xdl/src/Web.js

```diff
@@ -70,6 +58,60 @@ export async function hasWebSupportAsync(projectRoot) {
   return isWebConfigured;
 }
 
+async function getMissingReactNativeWebPackagesMessageAsync(projectRoot) {
+  const { pkg } = await readConfigJsonAsync(projectRoot);
+  const dependencies = await getMissingReactNativeWebPackagesAsync(projectRoot, pkg);
+  if (dependencies.length) {
+    const commandPrefix = isUsingYarn(projectRoot) ? 'yarn add ' : 'npm install ';
+    const command = commandPrefix + dependencies.join(' ');
+    return {
+      message: `* Install these packages: ${chalk.yellow(command)}`,
+      command,
+      dependencies,
+    };
+  }
+}
+
+function formatPackage({ name, version }) {
+  if (!version || version === '*') {
+    return name;
+  }
+  return `${name}@${version}`;
+}
+
+async function getMissingReactNativeWebPackagesAsync(projectRoot, appPackage) {
+  const { dependencies = {} } = appPackage;
+  const requiredPackages = [
+    { name: 'react', version: '*' },
+    { name: 'react-dom', version: '*' },
+    { name: 'react-native-web', version: '^0.11.2' },
+    { name: 'expo', version: '^33.0.0-alpha.web.1' },
```

Comment:
Note that this depends on the SDK version used. If write the versions here, we'll have to release a new version of Expo CLI whenever there's a new version that isn't covered by the semver range and users will also have to update to it to use that SDK version.

I think you could add `react-dom` and `react-native-web` to the version endpoint (https://exp.host/--/api/v2/versions), which already includes `react` and `expo` versions and is versioned by SDK version. You can use `Versions.sdkVersionsAsync()` in XDL to fetch them.

Reply:
Also this code probably should go in the `Doctor` module in XDL, which handles checks like this.